resources/texts/change_log.txt

```diff
@@ -46,6 +56,12 @@
 - Updated Japanese translations by @h1data
 - Brazilian Portuguese by @Patola
 
+* Known issues
+- This second beta introduces a change that affects the UI so please use buttons, dropdowns, menus, and flows. We hope we did this correctly but it is a tricky change. If you see the crash please report it to us, and we will also see it show up in our crash tracking tool. 
+- There is a slight regression in terms of seam alignment in different strategies we hope to resolve before stable
+- We see an increase in micro-segments in the outer wall, which we hope to resolve before stable (an increase in microsegments in the infill is expected)
```

Comment:
```suggestion
- We see an increase in micro-segments in the outer wall, which we hope to resolve before stable 
```

That's because there is no infill otherwise -- the previous behaviour was just wrong, so we don't really have to mention it.